### PR TITLE
Update Methods Section

### PR DESCRIPTION
**Pull request by cgreene**
This is a work in progress - filing so that @danich1 might be able to start on some of the content identified here.

### CODE REVIEW

**Comment by cgreene** [comment]
Ok @danich1  - I removed the WIP from the title because this is probably ready for a first look from you. Definitely not ready to merge though! 😁

**Review by danich1 — APPROVED** [review]
Answered/filled in the brackets with suggested changes. Will give the approval status, but won't merge until I get final approval from you.

File: content/03.methods.md
Comment (cgreene) [reply content/03.methods.md:111]: I thought we just set this to 10?

Comment (danich1) [reply content/03.methods.md:115]: Ack you are correct. Will update to reflect changes.

File: content/03.methods.md
Comment (danich1) [inline content/03.methods.md:129]:
```suggestion
We trained this model on document embeddings of PMCOA articles that did not contain a matching preprint version.
We used the following parameters to train the model: a hidden size of 2, a learning rate of 0.001, lambda_b of 0, lambda_c of 0.001, and lambda_d of 0.001 for 2000 iterations.
```

File: content/03.methods.md
Comment (dongbohu) [inline content/03.methods.md:125]:
```
We use k-d Trees in sklearn to search the papers. In order to make it more memory efficient, the k-d trees were sharded into four pieces and constructed beforehand. They are loaded into memory directly when the web server starts.
```
Do you need more details?

File: content/03.methods.md
Comment (danich1) [inline content/03.methods.md:125]:
```suggestion
We pass this representation onto our journal venue classifiers to identify journals based on the ten closest neighbors of individual papers as well as journal centroids.
Our classifiers use k-d trees in sklearn to efficiently search for similar papers and journals. 
We sharded the k-d trees into four separate pieces to make our models more memory efficient. 
We load each piece directly into memory as soon as the web server starts.
```

File: content/03.methods.md
Comment (cgreene) [inline content/03.methods.md:16]:
```suggestion

#### PubMed Central Open Access Corpus
```

File: content/03.methods.md
Comment (cgreene) [inline content/03.methods.md:58]:
```suggestion
Words absent from the word2vec model were ignored.

```

**Review by cgreene — COMMENTED** [review]
header changes

**Comment by cgreene** [comment]
If this builds let's merge it and move on to the next one 🚀

**Pull request merged**
